### PR TITLE
Rework application relationship

### PR DESCRIPTION
**Pull request by islean**
## Description

Working with the new order type application table is annoying - when using application.order_types one gets back a list of database entries when in reality one always want a list of order types; ideally a modifiable list. This PR changes it so that application.order_types is a modifiable list.

### Added

-

### Changed

- application.order_types is a modifiable list

### Fixed

-

### How to prepare for test

- [ ] Ssh to relevant server (depending on type of change)
- [ ] Use stage: `us`
- [ ] Paxa the environment: `paxa`
- [ ] Install on stage (example for Hasta):
    ```shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_cg -t cg -b [THIS-BRANCH-NAME] -a
    ```

### How to test

- [ ] Do ...

### Expected test outcome

- [ ] Check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review

- [ ] Tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan

- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

**Comment by islean** [comment]
Tested deploying to the VM and showing order types and manipulating in the web ui still works

File: cg/store/models.py [hunk 1]
```diff
@@ -181,10 +181,23 @@ class Application(Base):
     pipeline_limitations: Mapped[list["ApplicationLimitations"]] = orm.relationship(
         back_populates="application"
     )
-    order_types: Mapped[list["OrderTypeApplication"]] = orm.relationship(
-        "OrderTypeApplication", back_populates="application"
+    order_type_applications: Mapped[list["OrderTypeApplication"]] = orm.relationship(
```
Comment (diitaz93) [inline cg/store/models.py:184]: don't you need an alembic migration for the renaming of this attribute?

Comment (islean) [reply cg/store/models.py:184]: Nope, relationships are only SQLAlchemy - the underlying database needs no change for this to work

**Review by diitaz93 — APPROVED** [review]
Looks good 👍 The only thing is that you might need an alembic migration altering the column that you renamed

File: cg/store/models.py (hunk 1 above)
Comment (diitaz93) [reply cg/store/models.py:184]: ok then nevermind

**Comment by islean** [comment]
Deployed to the VMs

**Pull request merged**